lib/checks/canFindDenominatorInNumerator.js

```diff
@@ -0,0 +1,35 @@
+const Node = require('../node');
+
+// Returns true if by adding a term you can simplify part of the function into an integer
+// e.g. (2x+1)/(2x+3) -> True
+// e.g. (2x+1)/(2x^2 + 3) -> False
+function canFindDenominatorInNumerator(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '/' ) {
+    return false;
+  }
+  if (node.args.length !== 2) {
+    return false;
+  }
+
+  let numerator = node.args[0];
+  let denominator = node.args[1];
+  if (Node.Type.isParenthesis(numerator)) {
+    numerator = numerator.content;
+  }
+  if (Node.Type.isParenthesis(denominator)) {
+    denominator = denominator.content;
+  }
+
+  if (numerator.op !== '+') {
+    return false;
+  }
+  if (!('args' in denominator)) {
```

Comment:
> actually I think with this we probably need to check that there's some sort of subtraction or addition somewhere in the fraction, so maybe set it to try to catch any of those scenarios. What do you think?

I think the best way to check this is to make sure that the numerator and denominator aren't equal. if they're both `x` then they're equal and there's nothing to do. but also if they're both `x+1` then this step doesn't apply, we can just cancel (that step's code is elsewhere). How's that sound?